yt/funcs.py

```diff
@@ -1169,3 +1169,31 @@ def handle_mks_cgs(values, field_units):
     except YTEquivalentDimsError as e:
         values = values.to_equivalent(e.new_units, e.base)
     return values
+
+def validate_3d_array(obj):
+    if not iterable(obj) or len(obj) != 3:
+        raise TypeError("Expected an array of size (1,3), received '%s' of "
+                        "length %s" % (str(type(obj)).split("'")[1], len(obj)))
+
+def validate_float(obj):
```

Comment:
I would be worth having a doctstring to this function, as it may not be obvious that, e.g., `(1, 'kpc')` is actually a `float`.

Reply:
done